src/memefactory/ui/meme_detail/page.cljs

```diff
@@ -472,8 +480,11 @@
           (if @tx-success?
             "Voted"
             "Vote Stank")]]]
-       [:div "You can vote with up to " (format/format-token (quot @balance-dank 1e18) {:token "DANK"})]
-       [:div "Token will be returned to you after revealing your vote."]])))
+       (if (> @balance-dank 0)
```

Comment:
Isn't @balance-dank BigNumber here? In that case would need to use `bn/>` function

Reply:
I don't believe it is BigNumber, tests show that it's working, and the original balance was showing correctly. There was some discussion a while back about how the values aren't stored in BigNumber format within SQLite, so I wonder if that's the reason.

Reply:
Might make sense to make the changes anyways, i'll throw up another commit

Reply:
yeah, I don't mean SQLite's BigNumber, but bignumber that comes from web3.js, from [district-ui-account-balances](https://github.com/district0x/district-ui-web3-account-balances) subscription